### PR TITLE
Add support for Amazon Textract

### PR DESCRIPTION
**Pull request by kvasukib**
*Issue #, if available:*

*Description of changes:*

Add support for Amazon Textract service.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-android-sdk-textract-test/src/androidTest/java/com/amazonaws/services/textract/AmazonTextractInstrumentedTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class AmazonTextractInstrumentedTest extends AWSTestBase {
```
Comment (minbi) [inline aws-android-sdk-textract-test/src/androidTest/java/com/amazonaws/services/textract/AmazonTextractInstrumentedTest.java]: *IntegrationTest

File: .circleci/config.yml [hunk 2]
```diff
@@ -1493,6 +1503,7 @@ workflows:
             # - kinesisvideo-archivedmedia
             # - kinesisvideo
             - mobile-client
+            - textract
```
Comment (minbi) [inline .circleci/config.yml]: This same line should be added to the release_sdk workflow

File: .idea/modules.xml [hunk 3]
```diff
@@ -63,11 +63,14 @@
...
+      <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-textract-test/aws-android-sdk-textract-test.iml" filepath="$PROJECT_DIR$/aws-android-sdk-textract-test/aws-android-sdk-textract-test.iml" />
       <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-transcribe/aws-android-sdk-transcribe.iml" filepath="$PROJECT_DIR$/aws-android-sdk-transcribe/aws-android-sdk-transcribe.iml" />
       <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-transcribe-test/aws-android-sdk-transcribe-test.iml" filepath="$PROJECT_DIR$/aws-android-sdk-transcribe-test/aws-android-sdk-transcribe-test.iml" />
       <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-translate/aws-android-sdk-translate.iml" filepath="$PROJECT_DIR$/aws-android-sdk-translate/aws-android-sdk-translate.iml" />
       <module fileurl="file://$PROJECT_DIR$/aws-android-sdk-translate-test/aws-android-sdk-translate-test.iml" filepath="$PROJECT_DIR$/aws-android-sdk-translate-test/aws-android-sdk-translate-test.iml" />
       <module fileurl="file://$PROJECT_DIR$/aws-sdk-android.iml" filepath="$PROJECT_DIR$/aws-sdk-android.iml" />
+      <module fileurl="file://$PROJECT_DIR$/s3.iml" filepath="$PROJECT_DIR$/s3.iml" />
```
Comment (minbi) [inline .idea/modules.xml]: Not sure what this is.

File: aws-android-sdk-textract-test/src/androidTest/java/com/amazonaws/services/textract/AmazonTextractInstrumentedTest.java (hunk 1 above)
Comment (kvasukib) [reply aws-android-sdk-textract-test/src/androidTest/java/com/amazonaws/services/textract/AmazonTextractInstrumentedTest.java]: Fixed

File: .circleci/config.yml (hunk 2 above)
Comment (kvasukib) [reply .circleci/config.yml]: Added

**Comment by kvasukib** [comment]
Added pom.xml for textract

**Pull request merged**